lib/ui/semantics.dart

```diff
@@ -872,6 +876,7 @@ base class _NativeSemanticsUpdateBuilder extends NativeFieldWrapperClass1 implem
     required double elevation,
     required double thickness,
     required Rect rect,
+    String? identifier,
```

Comment:
why nullable? can this be required?

Reply:
tbh I'm not sure. I took inspiration from `tooltip` which is also a raw string, not `List<StringAttribute>`, and it is nullable.

But in the native semantics layer (at least on Android) the node's `resource-id` cannot be null, only empty.

Reply:
Done in c50bceec24433b36860ba82a1388f35fbdd0dbff.

Should I do the same for `tooltip` (probably in a separate PR)?